README.md

```diff
@@ -363,14 +382,14 @@ decorator(modelItem, inputs, state)
 
 `modelItem` is the list of font metadata fields provided by the underlying local
 fonts (or web fonts). `inputs` and `state`
-are specific to the *ArcsJs* system. _Inputs_ are data provided by a ArcsJs _
+are specific to the _ArcsJs_ system. _Inputs_ are data provided by a ArcsJs _
 Store_ connected to your Particle by the _Recipe_. `state` is an object
 maintained by the ArcsJs system that allows you to store and reference global
 data. The ArcsJs system is reactive, so anytime `inputs` or `state` changes,
 your particle is re-executed to update the UI.
 
 Ordinarily, `state` is globally mutable and readable, however this would be
-dangerous in a *Non Permissive Particle* because it could accumulate global font
+dangerous in a _Non Permissive Particle_ because it could accumulate global font
```

Comment:
We hyphenate this ("Non-Permissive Particle") several times earlier in the doc, we should be consistent one way or the other.